### PR TITLE
Add CSTG tests

### PR DESCRIPTION
**Pull request by mcollins-ttd**
This pull request adds tests for CSTG. They cover the happy path and the case of an invalid origin.

### CODE REVIEW

File: src/test/java/app/component/Operator.java [hunk 1]
```diff
@@ -182,6 +201,125 @@ public JsonNode v2TokenGenerateUsingPayload(String payload, boolean asOldPartici
         return v2DecryptEncryptedResponse(encryptedResponse, envelope.nonce(), getClientApiSecret(asOldParticipant));
     }
 
+    public JsonNode v2ClientSideTokenGenerate(String requestBody, boolean useValidOrigin) throws Exception {
+        final byte[] serverPublicKeyBytes = base64ToByteArray(CLIENT_SIDE_TOKEN_GENERATE_SERVER_PUBLIC_KEY.substring(PUBLIC_KEY_PREFIX_LENGTH));
```
Comment (gmsdelmundo) [inline src/test/java/app/component/Operator.java:205]: Not a big issue, but `final` isn't really needed for local variables

File: src/test/java/app/component/Operator.java [hunk 2]
```diff
@@ -52,12 +63,20 @@ public String toString() {
...
     }
 
+    private static final SecureRandom SECURE_RANDOM = new SecureRandom();
     private static final String CLIENT_API_KEY = EnvUtil.getEnv("UID2_E2E_API_KEY");
     private static final String CLIENT_API_SECRET = EnvUtil.getEnv("UID2_E2E_API_SECRET");
     private static final String CLIENT_API_KEY_BEFORE_OPTOUT_CUTOFF = EnvUtil.getEnv("UID2_E2E_API_KEY_OLD");
     private static final String CLIENT_API_SECRET_BEFORE_OPTOUT_CUTOFF = EnvUtil.getEnv("UID2_E2E_API_SECRET_OLD");
+    private static final String CLIENT_SIDE_TOKEN_GENERATE_SUBSCRIPTION_ID = EnvUtil.getEnv("UID2_E2E_SUBSCRIPTION_ID");
+    private static final String CLIENT_SIDE_TOKEN_GENERATE_SERVER_PUBLIC_KEY = EnvUtil.getEnv("UID2_E2E_SERVER_PUBLIC_KEY");
+    private static final String CLIENT_SIDE_TOKEN_GENERATE_ORIGIN = EnvUtil.getEnv("UID2_E2E_ORIGIN");
```
Comment (gmsdelmundo) [inline src/test/java/app/component/Operator.java:73]: Do both valid and invalid origins have to be passed in dynamically? Can they be hardcoded instead?

File: src/test/java/app/component/Operator.java [hunk 3]
```diff
@@ -182,6 +201,125 @@ public JsonNode v2TokenGenerateUsingPayload(String payload, boolean asOldPartici
...
+        }
+        final ECGenParameterSpec ecParameterSpec = new ECGenParameterSpec("secp256r1");
+        try {
+            keyPairGenerator.initialize(ecParameterSpec);
+        } catch (InvalidAlgorithmParameterException e) {
+            throw new RuntimeException(e);
+        }
+        return keyPairGenerator.genKeyPair();
+    }
+
```
Comment (jon8787) [inline src/test/java/app/component/Operator.java:241]: As @gmsdelmundo mentioned, any way to share this code with Operator (test) code?

File: src/test/java/app/component/Operator.java (hunk 1 above)
Comment (gmsdelmundo) [reply src/test/java/app/component/Operator.java:205]: Yep that's fine too, I don't have very strong opinions on either

**Review by gmsdelmundo — APPROVED** [review]

**Pull request merged**